stream_alert_cli/athena/handler.py

```diff
@@ -41,25 +41,7 @@
 MAX_QUERY_LENGTH = 262144
 
 
-def create_database(athena_client):
-    """Create the 'streamalert' Athena database
-
-    Args:
-        athena_client (boto3.client): Instantiated CLI AthenaClient
-    """
-    if athena_client.check_database_exists():
-        LOGGER_CLI.info('The \'streamalert\' database already exists, nothing to do')
-        return
-
-    create_db_success, create_db_result = athena_client.run_athena_query(
-        query='CREATE DATABASE streamalert')
-
-    if create_db_success and create_db_result['ResultSet'].get('Rows'):
-        LOGGER_CLI.info('streamalert database successfully created!')
-        LOGGER_CLI.info('results: %s', create_db_result['ResultSet']['Rows'])
-
-
-def rebuild_partitions(athena_client, options, config):
+def rebuild_partitions(options, config):
```

Comment:
This method is still needs the constraint on `options.type == data`, since it won't work for the `alerts` table